docs/getting-started.txt

```diff
@@ -27,75 +28,117 @@ code.
 Software
 ========
 
--  Determined agent and master nodes must be configured with Ubuntu 16.04 or higher, CentOS 7, or
-   macOS 10.13 or higher.
+-  Determined *agent* and *master* nodes must be configured with Ubuntu 16.04 or higher, CentOS 7,
+   or macOS 10.13 or higher.
 
 -  Agent nodes must have Docker installed.
 
--  To run jobs with GPUs, Nvidia drivers, version 384.81 or higher, must be installed on each agent.
-   Nvidia drivers can be installed as part of a CUDA installation but the rest of the CUDA toolkit
-   is not required.
+-  To run jobs with GPUs, install Nvidia drivers, version 384.81 or higher, on each agent. The
+   drivers can be installed as part of a CUDA installation but the rest of the CUDA toolkit is not
+   required.
 
 Hardware
 ========
 
--  At least 4-CPU cores are recommended for the master node, Intel Broadwell or later, and 8GB RAM,
-   and 200GB of free disk space. The master node does not require GPUs.
+-  Master node:
+
+   -  At least 4-CPU cores, Intel Broadwell or later. The master node does not require GPUs.
+   -  8GB RAM
+   -  200GB of free disk space.
+
+-  Agent Node:
 
--  at least 2-CPU cores are recommended for each agent node, Intel Broadwell or later, and 4GB RAM,
-   and 50GB of free disk space. If you are using GPUs, Nvidia GPUs with compute capability 3.7 or
-   greater are required, such as K80, P100, V100, A100, GTX 1080, GTX 1080 Ti, TITAN, and TITAN XP.
+   -  At least 2-CPU cores, Intel Broadwell or later.
+   -  If you are using GPUs, Nvidia GPUs with compute capability 3.7 or greater are required: K80,
+      P100, V100, A100, GTX 1080, GTX 1080 Ti, TITAN, or TITAN XP.
+   -  4GB RAM
+   -  50GB of free disk space.
 
 Docker
 ======
 
-For the examples that use an agent, install Docker to run containerized workloads. If you do not
-already have Docker installed, follow the :doc:`Installing Docker
+For the distributed example that uses an agent, install Docker to run containerized workloads. If
+you do not already have Docker installed, follow the :doc:`Installing Docker
 </sysadmin-deploy-on-prem/requirements>` instructions to install and run Docker on Linux or macOS.
 
-Training Example
-================
+Environment Variables
+=====================
+
+To run Determined on a remote master instance, set the remote IP address and port number in the
+``DET_MASTER`` environment variable:
```

Comment:
~To run Determined on a remote master instance~ To connect to a Determined master running on a remote instance

Reason: setting environment variables locally will not cause a Determined master process to start running on any remote instance.

Reply:
From a doc-layout perspective, I think it's weird to have a layout like this:

- Prerequisites
  - Environment variables (for remotely-running master)
- Install and Deploy Determined Locally
- ...

It seems weird because the envvar section is listed as a prereq, which implicitly assumes the master is already running remotely as a prereq.  But then the first step is "install and deply determined locally".

Perhaps a more correct layout would be:

- Prerequisites:
  - ...
- Install and Deploy Determined...
  - Option 1: ...on the Local Machine
  - Option 2: ...on a Remote Machine
  
     In the body of this subsection, we include the the content from the Environment Variables section

Reply:
Yeah, that's how it was originally but a prerequisite is a prerequisite. Presumably, the reader is going to do all 3 examples. I prefer your first option, which clarifies when the envar applies.

Reply:
I didn't give two options, I showed what you have, then I should what I proposed.  The original was:

- Prerequeisites
  - ...
- Install and Deploy Determined Locally
- Use a Remote Determined Instance

The indentation of the hierarchies is not the same across between what was originally there and what I proposed.  In the original version installing locally and connecting to a remote installation are consecutive steps, which is confusing.

I'm ok with installing and deploying being part of prerequisites, or as the first step after the prereq.  But the hierarchy should show that only one needs to be executed in any case.

Reply:
I think I effectively changed this as you suggested.